modules/LinearBoltzmannSolver/IterativeOperations/lbs_compute_fission_production.cc

```diff
@@ -30,9 +30,8 @@ double SteadySolver::ComputeFissionProduction(const std::vector<double>& phi)
       const double IntV_ShapeI = cell_matrices.Vi_vectors[i];
 
       //=============================== Loop over groups
-      //TODO: This should be nu
       for (size_t g = first_grp; g <= last_grp; ++g)
-        local_production += xs.nu_prompt_sigma_f[g] *
+        local_production += xs.nu_sigma_f[g] *
```

Comment:
i suppose this routine is in LB steady state. so nu.sigma is good. I see that before it had nu_prompt.sigma. I think it is more readable like this now